src/fcntl.rs

```diff
@@ -825,6 +825,10 @@ pub enum FcntlArg<'a> {
     /// Both descriptors must reference regular files in the same volume.
     #[cfg(apple_targets)]
     F_TRANSFEREXTENTS(RawFd),
+    /// Set or clear the read ahead amount for sequential access or
+    /// disable it with 0 or to system default for any value < 0.
+    #[cfg(target_os = "freebsd")]
+    F_READAHEAD(c_int),
```

Comment:
Is this something related to the kernel page cache, something used to control the pre-fetch strategy?

Reply:
it manages file data caching yes. will adjust the comment.